examples/helm/index.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as helm from "@pulumi/kubernetes/helm";
+
+const nginx = new helm.v2.Chart("simple-nginx", {
+  // Represents chart `stable/nginx-lego@v0.3.1`.
+  repo: "stable", chart: "nginx-lego", version: "0.3.1",
```

Comment:
Curious - any particular reason to break these out vs. just accept as a single string?  I feel like I always see these mentioned by `stable/jenkins`.

Reply:
Parsing the repo name out of that ID is hard in general. This ID can be 5 things (URL, path, well-known ID, etc.)

Reply:
Why do we need to parse it out?   The only place I see it being used these two pieces get concatenated together.

Reply:
I think we've agreed offline to keep this as is.